src/fdcache.cpp

```diff
@@ -2577,6 +2583,77 @@ ssize_t FdEntity::Write(const char* bytes, off_t start, size_t size)
   return wsize;
 }
 
+// [NOTE]
+// The meta(headers) cannot be updated while multipart upload is in progress.
+// Therefore, this method holds the header that will be updated if a multipart
+// upload is in progress and returns true.
+// All pending headers are updated after the multipart upload is complete.
+// Only the specified headers are retained as meta. Other headers are merged
+// when updated.
+//
+bool FdEntity::IsPendingMeta(const headers_t& meta, const metakeys_t& keys)
```

Comment:
Hi， ggtakec, this method combines two function, judge and append headers to pending_headers，can this method split to two function for easy understand: IsUploading and UpdatePendingData